res/controllers/engine-api.d.ts

```diff
@@ -301,4 +301,57 @@ declare namespace engine {
      *               SoftStart with low factors would take a while until sound is audible. [default = 1.0]
      */
     function softStart(deck: number, activate: boolean, factor?: number): void;
+
+    enum WellKnownCharsets {
+        Latin1,
+        ISO_8859_1,
+        Latin9,
+        ISO_8859_15,
+        UCS2,
+        ISO_10646_UCS_2
+    }
+
+    /**
+     * Converts a string into another charset.
+     *
+     * This function is useful to display text on a device that does not make use of UTF-8.
+     * Available charset names are listed here: http://www.iana.org/assignments/character-sets/character-sets.xhtml
+     *
+     * Depending on the system you run on, this function may convert characters that are unsupported by the target
+     * charset to either the codepoint for '?' or the substitute character.
+     * @see https://en.wikipedia.org/wiki/Substitute_character
+     *
+     * Example usage:
+     *
+     *         const group = "…"  // device midi group
+     *         const titleLength = engine.getValue(group, "track_title_length")
```

Comment:
Unless I'm wrong, `track_title_length` and `track_title_x` are undocumented. Should I document them here?

Reply:
Are those even merged? Can you link me to the PR? I highly oppose those COs...